docs/vc-spec.md

```diff
@@ -1,6 +1,163 @@
-# II Verifiable Credential Spec
+# II Verifiable Credential Spec (MVP)
+
+
+## Issuer API
+
+This section describes the (Candid) interface to be implemented by an issuer of verifiable credentials on the IC.
+This interface is used by the II-canister during attribute sharing flow (cf. [flow description](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/attribute-sharing.md))
+An example implementation of the interface is given in [demos/vc_issuer](../demos/vc_issuer).
+
+The Candid interface is as follows, and the subsequent sections describe the
+services and the corresponding messages in more detail.
+
+```candid
+type CredentialSpec = record { info : text };
+type GetCredentialRequest = record {
+    signed_id_alias : SignedIdAlias;
+    prepared_context : opt blob;
```

Comment:
Is this field `blob` or `text`? Further down, it is defined as `text`.

Reply:
`blob`, fixed now.